package.json

```diff
@@ -16,7 +16,8 @@
     "test": "jest",
     "lint": "eslint src/ test/",
     "lintfix": "eslint --fix src/ test/",
-    "publish-docs": "npm run docs && gh-pages -d docs"
+    "publish-docs": "npm run docs && gh-pages -d docs",
+	"prepare": "npm run build"
```

Comment:
formatting seems a little funky here

Reply:
ah, my edit made this a tab instead of spaces - fixed.